android/java/org/chromium/chrome/browser/ntp/BraveNewTabPageLayout.java

```diff
@@ -11,22 +11,27 @@
 import android.app.Activity;
 import android.content.Context;
 import android.content.Intent;
+import android.content.SharedPreferences;
 import android.content.pm.ActivityInfo;
 import android.content.res.ColorStateList;
 import android.content.res.Configuration;
 import android.graphics.Bitmap;
 import android.graphics.Color;
 import android.graphics.Point;
+import android.graphics.Rect;
 import android.graphics.drawable.BitmapDrawable;
 import android.net.Uri;
 import android.os.Build;
 import android.os.Bundle;
 import android.os.Handler;
+import android.os.Looper;
```

Comment:
these seems unused:
```
//import android.os.Bundle;
//import android.os.Looper;
//import android.util.Base64;
//import android.util.Pair;
//import android.view.ViewStub;
//import androidx.annotation.Nullable;
//import androidx.core.view.ViewCompat;
//import androidx.core.widget.NestedScrollView;
//import androidx.preference.PreferenceManager;
//import org.chromium.base.TraceEvent;
//import org.chromium.brave_news.mojom.Deal;
??import org.chromium.brave_news.mojom.Feed;
??import org.chromium.brave_news.mojom.Image;
//import org.chromium.brave_news.mojom.PromotedArticle;
//import org.chromium.brave_news.mojom.Publisher;
//import org.chromium.chrome.browser.ChromeTabbedActivity;
//import org.chromium.chrome.browser.compositor.layouts.OverviewModeBehavior;
//import org.chromium.chrome.browser.sync.settings.BraveManageSyncSettings;
//import org.chromium.chrome.browser.tab.EmptyTabObserver;
//import org.chromium.chrome.browser.tab.TabObserver;
//import org.chromium.chrome.browser.tabmodel.TabModel;
??import org.chromium.url.mojom.Url;
//import java.util.Collections;
import java.util.Locale;
//import java.util.concurrent.atomic.AtomicInteger;
```
though not all mentioned are from the current PR

Reply:
removed

Reply:
:+1: